src/com/timoschwarzer/hkmodinstaller/data/ModBundle.java

```diff
@@ -31,15 +38,18 @@ public ModBundle(String filename) throws Exception {
         load();
     }
 
+    /** Ensures and loads all elements of the ModBundle from the filename given */
     private void load() throws Exception {
+        // Pre-initialization and version validation
         ZipEntry manifest = zipFile.getEntry("mod.json");
 
-        JSONObject manifestObject = new JSONObject(IOUtils.toString(zipFile.getInputStream(manifest)));
+        JSONObject manifestObject = new JSONObject(IOUtils.toString(zipFile.getInputStream(manifest), Charset.defaultCharset()));
```

Comment:
I don't think using the default OS charset here is a good idea. Imagine one has created a modbundle on a UNIX machine (UTF-8 charset) and opens it on a Windows machine (Windows 1252).
Could you replace `Charset.defaultCharset()` with `StandardCharsets.UTF_8`, please? :smile: